README.md

```diff
@@ -1,73 +1,83 @@
 # Tarantool
 
-[![Build Status][travis-badge]][travis-url]
-[![Build Status][gitlab-ci-badge]][gitlab-ci-url]
+[![Actions Status][actions-badge]][actions-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
-
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
+https://www.tarantool.io/en/
```

Comment:
I would remove this link as it looks lonely and not tied to anything. Better, make `Tarantool` from the follow-up sentence clickable:

```diff
- Tarantool is an ...
+ [Tarantool][tarantool-url] is an ...
```

Reply:
Good idea. Changed.